.semaphore/push-images/alp.yml

```diff
@@ -18,12 +18,6 @@ global_job_config:
   prologue:
     commands:
       - checkout
-      # Semaphore mounts a copy-on-write FS as /var/lib/docker in order to provide a pre-loaded cache of
-      # some images. However, the cache is not useful to us and the copy-on-write FS is a big problem given
-      # how much we churn docker containers during the build.  Disable it.
-      - sudo systemctl stop docker
-      - sudo umount /var/lib/docker && sudo killall qemu-nbd || true
```

Comment:
On f1 generation VMs, a third disk `/dev/sdc1` is mounted under `/var/lib/docker`. It has 100GB of storage space and pre-loaded images from semaphore.

```
$ mount | grep sdc
/dev/sdc1 on /var/lib/docker type ext4 (rw,noatime,nobarrier)
```

Once unmounted, we are using the first 2 physical volumes (`/dev/sda3` ~25GB + `/dev/sdb1` ~30GB) for Calico image caches. This is not ideal because `/dev/sdc1` is much bigger.